ledger-tool/src/ledger_utils.rs

```diff
@@ -237,7 +236,7 @@ pub fn load_bank_forks(
             None,
             None, // Maybe support this later, though
             accounts_update_notifier,
-            &Arc::default(),
+            &exit,
```

Comment:
Had to click through quite the rabbit hole, but I see this gets hooked up to `AccountsIndex` which has some threads that live on so nice to have this hooked up.